src/tensor_operations/matrix_decomposition.jl

```diff
@@ -590,6 +590,19 @@ function sqrt_decomp(D::ITensor, u::Index, v::Index)
   return sqrtDL, prime(δᵤᵥ), sqrtDR
 end
 
+# Take the square root of T assuming it is Hermitian
+# TODO: add more general index structures
+function Base.sqrt(T::ITensor; ishermitian=true, atol=1e-15)
```

Comment:
```suggestion
function Base.sqrt(T::ITensor; ishermitian=true, atol=eps(real(eltype(T)))
```
That is more general to other element types (like single precision).